src/lib/actions/analytics.ts

```diff
@@ -38,21 +41,27 @@ function plausible(domain: string): AnalyticsPlugin {
     };
 }
 
+const PLAUSIBLE_DOMAINS = {
+    CLOUD: 'cloud.appwrite.io',
+    GLOBAL: 'console.appwrite',
+    SELF_HOSTED: 'self-hosted.appwrite'
+};
+
 const analytics = Analytics({
     app: 'appwrite',
-    plugins:
-        isCloud && browser
-            ? [
-                  plausible('cloud.appwrite.io'),
-                  googleTagManager({
-                      containerId: [VARS.GOOGLE_TAG || 'GTM-P3T9TBV']
-                  })
-              ]
-            : [
-                  googleAnalytics({
-                      measurementIds: [VARS.GOOGLE_ANALYTICS || 'G-R4YJ9JN8L4']
-                  })
-              ]
+    plugins: isCloud
```

Comment:
Browser is no longer needed?

Reply:
I moved it to the plugin, kinda annoyed me there.